assets/scss/drupal/module/_civiremote_funding.scss

```diff
@@ -27,4 +27,12 @@ $legend-font-size: 1.5rem;
       margin-top: 0;
     }
   }
+
+  // Required, if module Form Tips is used.
+  // https://www.drupal.org/project/formtips
+  // If z-index of .custom-control is set to a concrete number, input fields
+  // of other controls overlap tool tips independent of the tool tips' z-index.
+  .custom-control {
+    z-index: initial !important;
```

Comment:
Really no way to avoid `!important` here as well?

Reply:
It's not applied without `!important`.

Reply:
As discussed, let's use a more specific selector instead of `!important`.